superset-frontend/src/dashboard/components/nativeFilters/types.ts

```diff
@@ -78,15 +82,15 @@ export type FilterType = 'text' | 'date';
 export interface Filter {
   allowsMultipleValues: boolean;
   cascadeParentIds: string[];
-  defaultValue: string | null;
-  currentValue?: (string | number | boolean)[] | null;
+  defaultValue: any;
+  currentValue?: any;
```

Comment:
Same here

Reply:
it's value that we are passing to the native filter, so it should be `currentValue` together with default value and not `currentState`

I think may be we need to discuss in future if we want to render some things like `Allow multiple selection` checkbox inside of native filter chart so it should be really changed to currentState but for now all filters working only with `currentValue`